latch_cli/main.py

```diff
@@ -194,7 +190,7 @@ def _display(row, style):
             _display(row, style)
 
 
-@click.command("local-execute")
+@main.command("local-execute")
 @click.argument("pkg_root", nargs=1, type=click.Path(exists=True))
 def local_execute(pkg_root: Path):
```

Comment:
may wanna remove the `Visit docs.latch.bio ...` at the bottom of this for consistency